.github/workflows/test-template.yml

```diff
@@ -102,7 +102,7 @@ jobs:
   # CLIENT (functional and legacy tests without types test)
   #
   client:
-    name: Client (functional and legacy tests without types test)
+    name: Client func/legacy notypes
```

Comment:
Do you want to parenthesize this like the others for consistency?

Reply:
I had the suggested the same here: https://github.com/prisma/prisma/pull/21365#pullrequestreview-1659302613

Reply:
We can't because we don't have the space in the UI

Reply:
Are you sure you are not zooming in or similar in your browser? 😆

Reply:
Yes 😄 

Zooming does not change anything, check https://github.com/prisma/prisma/actions/runs/6429098126